### PR TITLE
Storybook v7

### PR DESCRIPTION
**Pull request by artmsilva**
## What I did

to do's
- [ ] add storybook modes https://www.chromatic.com/docs/modes/
- [ ] add theming https://www.chromatic.com/docs/themes/

### CODE REVIEW

File: .storybook/preview.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        ...Object.keys(theme.colors).map((color) => ({
+          name: color,
+          value: theme.colors[color].value,
+        })),
+      ],
+    },
+    viewport: {
+      viewports: INITIAL_VIEWPORTS,
+    },
+    darkMode: {
```
Comment (wp-aberg) [inline .storybook/preview.ts:39]: How can we toggle this? I don't see a menu item or button in the UI?

**Review by wp-aberg — COMMENTED** [review]
I think having the side by side Light and Dark mode version of the component was very helpful during development. Even with modes I think it's helpful to have that immediate feedback on when you may have forgotten to add a necessary class.

File: .storybook/main.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+function getAbsolutePath(value: string): any {
+  return dirname(require.resolve(join(value, "package.json")));
+}
+const config: StorybookConfig = {
+  stories: ["../ui/(**|!(node_modules))/src/*.stories.@(ts|tsx|js|jsx)"],
+  addons: [
+    getAbsolutePath("@storybook/addon-links"),
+    getAbsolutePath("@storybook/addon-essentials"),
+    getAbsolutePath("@storybook/addon-onboarding"),
+    getAbsolutePath("@storybook/addon-interactions"),
```
Comment (wp-aberg) [inline .storybook/main.ts:18]: a11y is an important one for us
`getAbsolutePath("@storybook/addon-a11y"),`

It's confusing to have addons in the package.json and not to have them here, if we really aren't using them we should pull them out.

**Comment by valentinpalkovic** [comment]
Hi @artmsilva,

My name is Valentin, and I am one of the Storybook Core Maintainers. I am currently working on the Storybook 8 RC and soon the release itself, and I randomly picked your project to try different things related to Storybook 8. 

I have stumbled upon this PR, and I am curious why it got abandoned. Do you miss some features in Storybook 7, or do you have any general hurdles that stop you from upgrading?

**Comment by artmsilva** [comment]
> Hi @artmsilva,
> 
> My name is Valentin, and I am one of the Storybook Core Maintainers. I am currently working on the Storybook 8 RC and soon the release itself, and I randomly picked your project to try different things related to Storybook 8.
> 
> I have stumbled upon this PR, and I am curious why it got abandoned. Do you miss some features in Storybook 7, or do you have any general hurdles that stop you from upgrading?

@valentinpalkovic I opened this PR as a spike. Our `v2` branch and v2 project is using Storybook v7. We will likely upgrade to v8 close to the end of the v2 project. Feel free to try out Storybook 8 RC in the `v2` branch.

**Pull request closed without merging**